RealmTasks Android/app/src/main/java/io/realm/realmtasks/list/TaskAdapter.java

```diff
@@ -41,12 +44,29 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
         if (task.isValid()) {
             final TextView text = itemViewHolder.getText();
             text.setText(task.getText());
+
+            Date taskDate = task.getDate();
+            if(taskDate != null) {
+                CharSequence naturalDateString = naturalDateFrom(taskDate);
+                itemViewHolder.setMetadataText(naturalDateString);
+            } else {
+                itemViewHolder.setMetadataText(null);
+            }
             narrowRightMargin(text);
             narrowRightMargin(itemViewHolder.getEditText());
             itemViewHolder.setCompleted(task.isCompleted());
         }
     }
 
+    private String naturalDateFrom(@NonNull Date taskDueDate) {
+        return DateUtils.getRelativeDateTimeString(
+                context,
+                taskDueDate.getTime(),
+                DateUtils.DAY_IN_MILLIS,
+                DateUtils.WEEK_IN_MILLIS, 0).toString()
+                .replace(",", " at");
```

Comment:
I don't like the replace here, but couldn't find a way to change the delimiter with getRelativeDateTimeString.  This works.  Certainly open to a suggestion to configure getRelativeDateTimeString to use at instead of comma.